### PR TITLE
feat(tabs): added responsive state for vertical tabs

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/3698

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
Looks good @mcoker

File: src/patternfly/components/Tabs/examples/Tabs.md [hunk 1]
```diff
@@ -205,3 +233,6 @@ Whenever a list of tabs is unique on the current page, it can be used in a `<nav
 | `.pf-m-current` | `.pf-c-tabs__item` | Indicates that a tab item is currently selected. |
 | `.pf-m-inset-{none, sm, md, lg, xl, 2xl}{-on-[md, lg, xl, 2xl]}` | `.pf-c-tabs` | Modifies the tabs component padding/inset to visually match padding of other adjacent components. |
 | `.pf-m-color-scheme--light-300` | `.pf-c-tabs` | Modifies the tabs component tab background colors. |
+| `.pf-m-expandable{-on-[breakpoint]}` | `.pf-c-tabs` | Modifies the tabs component to be expandable via a toggle. **Note:** works with vertical tabs only. |
+| `.pf-m-non-expandable{-on-[breakpoint]}` | `.pf-c-tabs` | Modifies the tabs component to be non-expandable. |
```
[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/components/Tabs/examples/Tabs.md:237]: `pf-m-non-expandable` doesn't seem like the ideal name, though we use `pf-m-static` for something that supports `pf-m-sticky`, but isn't sticky... open to more ideas though!

**Comment by mcoker** [comment]
@mattnolting updated to show vertical overflow and added the expanded toggle margin.

**Comment by mcoker** [comment]
@mattnolting an aside, do you know why we set `overflow: hidden` to begin with?

**Comment by mattnolting** [comment]
> @mattnolting an aside, do you know why we set overflow: hidden to begin with?

@mcoker, no I don't, I assume it was a bit of legacy code ¯\_(ツ)_/¯

**Review by mattnolting — APPROVED** [review]
Looks great, nice work!

**Pull request merged**